src/Chainweb/Pact/NoCoinbase.hs

```diff
@@ -8,7 +8,7 @@
 -- Maintainer: Lars Kuhtz <lars@kadena.io>
 -- Stability: experimental
 --
--- TODO
+-- A fake coin base for testing purposes.
```

Comment:
It's not fake - it's necessary for genesis transactions

Reply:
Oh, I didn't know that. I fixed the comment.